template/src/main.ts

```diff
@@ -2,17 +2,18 @@ import Vue from 'vue';
 import VueRouter from 'vue-router';
 
 import './sass/main.scss';
+import {NavbarComponent} from './components/navbar';
```

Comment:
Minor - can you add spaces within the angle brackets?

Reply:
yes. It was the webstorm flavour. :D